virttest/qemu_monitor.py

```diff
@@ -22,13 +23,17 @@
     logging.warning("Could not import json module. "
                     "QMP monitor functionality disabled.")
 
-from . import passfd_setup
 from . import utils_misc
 from . import cartesian_config
 from . import data_dir
 
 from virttest.qemu_capabilities import Flags
 
+if sys.version_info[:2] >= (3, 3):
+    import array
+else:
+    import passfd
```

Comment:
What I meant by defining our sendmsg is that we could do here:

```python
if sys.version[0] >= 3:
    import array
    def sendmsg(socket, cmd, fd):
        return socket.sendmsg([b"%s\n" % cmd.encode()], [(socket.SOL_SOCKET, socket.SCM_RIGHTS, array.array("i", [fd]))])
else:
    import passfd
    def sendmsg(socket, cmd, fd):
        return passfd.sendfd(socket, fd, b"%s" % cmd.encode()])
```

and than only use:

    sendmsg(self._socket, cmd, fd)